resources/assets/components/InboxItem/InboxTile.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+class InboxTile extends React.Component {
+  displayImage(photo_url) {
```

Comment:
i wonder if this should be some sort of helper like `getImageUrLFromProp` or something. cause I imagine there could be more than one place we would want to parse the default image to a particular image